### PR TITLE
SD-1875 Implement integrated logging for Superdesk

### PR DESCRIPTION
**Pull request by mugurrus**

### CODE REVIEW

**Comment by sivakuna-aap** [comment]
:+1:

File: server/settings.py [hunk 1]
```diff
@@ -39,6 +39,9 @@ def env(variable, fallback_value=None):
 DATE_FORMAT = '%Y-%m-%dT%H:%M:%S+0000'
 PAGINATION_LIMIT = 200
 
+LOG_SERVER_ADDRESS = 'localhost'
```
Comment (amagdas) [inline server/settings.py]: We should read these settings from Sys env.

**Comment by amagdas** [comment]
:+1:

**Comment by akintolga** [comment]
:+1:

File: docker/kibana/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# import base image
+FROM ubuntu:trusty
+
+# Dependencies
+RUN apt-get update -qq
+RUN sudo apt-get install -y -qq nginx-full wget vim
```
Comment (amagdas) [inline docker/kibana/Dockerfile]: Why is vim needed?

File: docker/kibana/kibana.yml [hunk 3]
```diff
@@ -0,0 +1,23 @@
+port: 5601
+host: "0.0.0.0"
+elasticsearch_url: "http://elastic:9200"
+elasticsearch_preserve_host: true
+kibana_index: ".kibana"
+default_app_id: "discover"
+request_timeout: 300000
```
Comment (amagdas) [inline docker/kibana/kibana.yml]: That's quite a big timeout.

**Comment by amagdas** [comment]
:+1:

**Pull request merged**